lib/linguist/heuristics.rb

```diff
@@ -491,5 +499,14 @@ def call(data)
         Language["XML"]
       end
     end
+
+    disambiguate ".user" do |data|
```

Comment:
I agree *.user is quite broad. Does it make sense to register it under Text as well in languages.yml and keep the XML disambiguation? In future, folks will add other (else) branches to this block for varied languages. If it incurs performance penalties in platform or there are other reasons, I am all for removing it. :)